src/Toolbox.php

```diff
@@ -2492,6 +2492,8 @@ public static function decodeArrayFromInput($value)
      * @since 0.84.2
      *
      * @return void  display error if not permit
+     *
+     * @FIXME Remove this in GLPI 10.1
```

Comment:
Shoudl not this be deprecated right now?

Reply:
This is just useless, calling it does not hurts, but having a deprecation message added could trigger additionnal support requests if any plugin is using the method.

Reply:
IK had in min the deprecated message was intended to do this, calls will just fatal fail when method will be removed. But anyways, I don't really care